index.html

```diff
@@ -6123,22 +6152,29 @@ <h3><dfn>Element Send Keys</dfn></h3>
   of <a>trying</a> to <a>get a known connected element</a>
   with <a>url variable</a> <var>element id</var>.
 
- <li><p><a>Scroll into view</a> the <var>element</var>.
+ <li><p>Let <var>file</var> be <code>true</code> if If <var>element</var> is
+   an <a><code>input</code> element</a> whose
+    <a><code>type</code> attribute</a> is <a>File</a>, or <code>false</code> otherwise.
```

Comment:
You should use `<a><code>input</code> element</a> in the <a>file upload state</a>`, and I think you should remove the above _file_ variable.

```suggestion
  an <a><code>input</code> element</a> in the <a>file upload state</a>,
  or false otherwise.
```

Reply:
I don't understand what you mean about removing the variable, we end up needing to use it in two places, so it seems senisble to keep that logic in a single place.

Reply:
Leave the variable if you want.